tempodb/tempodb.go

```diff
@@ -545,13 +546,13 @@ func (rw *readerWriter) pollBlocklist() {
 }
 
 // includeBlock indicates whether a given block should be included in a backend search
-func includeBlock(b *backend.BlockMeta, _ common.ID, blockStart []byte, blockEnd []byte, timeStart int64, timeEnd int64) bool {
+func includeBlock(b *backend.BlockMeta, id common.ID, blockStart []byte, blockEnd []byte, timeStart int64, timeEnd int64) bool {
 	// todo: restore this functionality once it works. min/max ids are currently not recorded
 	//    https://github.com/grafana/tempo/issues/1903
 	//  correctly in a block
```

Comment:
Is this still true?

Reply:
No, it works now (fixed in #2867) which is good because traceql metrics and this new compactor rely on it :)  Removed the obsolete comment.